python/cudf/io/csv.py

```diff
@@ -233,6 +236,9 @@ def read_csv(filepath_or_buffer, lineterminator='\n',
     if thousands == delimiter:
         raise ValueError("thousands cannot be the same as delimiter")
 
+    if nrows and skipfooter != 0:
```

Comment:
`nrows` can technically be 0 as a valid value to return an empty dataframe with column names and dtypes.